src/mugs.js

```diff
@@ -94,16 +202,13 @@ define([
          * `fixSerializationWarnings` to automatically fix the warnings.
          */
         getSerializationWarnings: function () {
-            var dupId = this.p.conflictedNodeId;
-            if (!_.isUndefined(dupId)) {
-                return [{
-                    code: "conflictedNodeId",
-                    message: "'" + dupId + "' has the same Question ID as " +
-                        "another question in the same group. Please choose " +
-                        "a unique Question ID."
-                }];
-            }
-            return [];
+            var warnings = [];
+            this.messages.each(function (msg) {
```

Comment:
minor: good candidate for underscore chaining. `_.chain(this.messages).filter().map().value()`. can be slightly more readable